tools/testoperator/src/tests/mod.rs

```diff
@@ -23,30 +23,33 @@ use test_framework::{
 
 pub(crate) mod bench;
 pub(crate) mod data_consistency;
-pub(crate) mod http_consistency;
+pub(crate) mod http;
 pub(crate) mod load;
 pub(crate) mod throughput;
-
+mod util;
 pub(crate) type RowCounts = BTreeMap<String, usize>;
 
-pub(crate) fn get_app_and_start_request(args: &CommonArgs) -> anyhow::Result<(App, StartRequest)> {
+pub(crate) fn get_app_and_start_request(
+    args: &CommonArgs,
+    data_dir: Option<&Path>,
+) -> anyhow::Result<(App, StartRequest)> {
```

Comment:
```suggestion
pub(crate) fn get_app_and_start_request(
    args: &CommonArgs,
) -> anyhow::Result<(App, StartRequest)> {
```

this doesn't need an additional function argument if data dir is in common args